input/profiles/StructureDefinition-qicore-medicationrequest.json

```diff
@@ -70,7 +70,27 @@
           }
         ]
       },
-      {
+	 {
+			"id" : "MedicationRequest.medicationAdherence",
+			"extension" : [
+			  {
+				"url" : "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-keyelement",
+				"valueBoolean" : true
+			  }
+			],
+			"path" : "MedicationRequest.extension",
+			"sliceName" : "medicationAdherence",
+			"short" : "URL: http://hl7.org/fhir/us/core/StructureDefinition/us-core-medication-adherence",
```

Comment:
same ... this needs not be a short. It needs to be rendered as a hyperlink to the actual us-core-medication-adherence extension.

Reply:
The qi-core medicationRequest and medicationNotRequested are derived from US Core MedicationRequest where the medicationAdherence looks like the following. I've made quite an effort to match the appearance and felt good about it. Obviously they are just snake feet.
![{DB94E611-3C67-4C4C-B050-B8FA635DA905}](https://github.com/user-attachments/assets/be067e0b-f83a-4613-85da-b29f2fedd756)